app/helpers/transition_period_helper.rb

```diff
@@ -0,0 +1,12 @@
+module TransitionPeriodHelper
+  SWITCHOVER_TIME = Time.zone.parse('2020-01-01 17:00:00').in_time_zone
```

Comment:
Date needs updating if we want to test this?

Reply:
sorry ran `git push` not `git push --force`